### PR TITLE
bug fix - move fd store time and mtu set

### PR DESCRIPTION
**Pull request by kailiangz1**

### CODE REVIEW

File: drivers/vdpa/virtio/rte_vf_rpc.c [hunk 1]
```diff
@@ -43,7 +43,9 @@ rte_vdpa_vf_dev_add(char *vf_name, const char *vm_uuid,
 		offset = snprintf(args, RTE_DEV_NAME_MAX_LEN, "%s=%s", VIRTIO_ARG_VDPA, VIRTIO_ARG_VDPA_VALUE_VF);
 
 	if (stage1)
-		snprintf(args + offset, RTE_DEV_NAME_MAX_LEN, ",%s=%s", VIRTIO_ARG_VDPA_STAGE, VIRTIO_ARG_VDPA_VALUE_STAGE);
+		offset += snprintf(args + offset, RTE_DEV_NAME_MAX_LEN, ",%s=%s", VIRTIO_ARG_VDPA_STAGE, VIRTIO_ARG_VDPA_VALUE_STAGE);
+
+	offset += snprintf(args + offset, MAX_PATH_LEN, ",%s=%s", VIRTIO_ARG_VDPA_SOCK_PATH, ifname);
```
Comment (Ch3n60x) [inline drivers/vdpa/virtio/rte_vf_rpc.c:48]: this 'offset +=' seems not needed as offset is not used later

File: drivers/vdpa/virtio/virtio_vdpa.c [hunk 2]
```diff
@@ -2034,6 +2018,14 @@ virtio_pci_devargs_parse(struct rte_devargs *devargs, int *vdpa, rte_uuid_t vm_u
 			DRV_LOG(ERR, "Failed to parse %s", VIRTIO_ARG_VDPA_STAGE);
 	}
 
+	if (rte_kvargs_count(kvlist, VIRTIO_ARG_VDPA_SOCK_PATH) == 1) {
+		/* stage mode selected, debug only
+		 */
```
Comment (Ch3n60x) [inline drivers/vdpa/virtio/virtio_vdpa.c]: above comment is wrong due to copy-paste

File: drivers/vdpa/virtio/rte_vf_rpc.c (hunk 1 above)
Comment (kailiangz1) [reply drivers/vdpa/virtio/rte_vf_rpc.c:48]: just in case we add another line , needed add + again.

File: drivers/vdpa/virtio/virtio_vdpa.c (hunk 2 above)
Comment (kailiangz1) [reply drivers/vdpa/virtio/virtio_vdpa.c]: Done

**Pull request merged**